kustomize/data-plane/prerequisites/service-mesh/resources/service-mesh-control-plane.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+  addons:
+    grafana:
```

Comment:
Just out of curiosity. Do we need these addons for data plane? These are focused on monitoring. If we need to speed up, I would suggest to remove it.

Reply:
Good question! I guess we can remove the grafana addon, not sure about jaeger (if it is required by istio or not). I will try to remove grafana and see if the deployment is successful.

The current time of the data plane deployment is ~ 1 minute 40 seconds which seems not to be critical.